tests/by-util/test_pmap.rs

```diff
@@ -284,8 +309,10 @@ fn assert_format(pid: u32, s: &str, show_path: bool) {
     let re = Regex::new(&format!("{base_pattern}{mapping_pattern}")).unwrap();
     assert!(re.is_match(memory_map));
 
-    let re = Regex::new("^ total +[1-9][0-9]*K$").unwrap();
-    assert!(re.is_match(last_line));
+    if !quiet {
+        let re = Regex::new("^ total +[1-9][0-9]*K$").unwrap();
+        assert!(re.is_match(last_line));
+    }
```

Comment:
You have to handle both the `if` and the `else` case. Otherwise, if `quiet` is `true`, you skip the assert without ensuring there is no "total" line. And thus the test passes even without the changes you made to `pmap.rs` and `pmap_config.rs` ;-)